pkg/sql/colexec/colexecbase/cast_test.go

```diff
@@ -48,7 +48,8 @@ func TestRandomizedCast(t *testing.T) {
 		for {
 			from, to = randgen.RandType(rng), randgen.RandType(rng)
 			if _, ok := cast.LookupCastVolatility(from, to); ok {
-				if colexecbase.IsCastSupported(from, to) {
+				// Skip identical casts since they are not interesting.
+				if colexecbase.IsCastSupported(from, to) && !colexecbase.IsIdentityCast(from, to) {
```

Comment:
I'm worried about losing this testing coverage. Do you have an explanation for why these casts are so slow? I'd expect them to be faster than any other casts.